README.md

```diff
@@ -301,45 +301,21 @@ You can use these commands to get more information about the experiment
-* [Customize your own Tuner](docs/en_US/Tuner/CustomizeTuner.rst)
-* [Implement customized TrainingService](docs/en_US/TrainingService/HowToImplementTrainingService.rst)
-* [Implement a new NAS trainer on NNI](docs/en_US/NAS/Advanced.rst)
-* [Customize your own Advisor](docs/en_US/Tuner/CustomizeAdvisor.rst)
-
-## **External Repositories and References**
-With authors' permission, we listed a set of NNI usage examples and relevant articles.
-
-* ### **External Repositories** ###
-   * Run [ENAS](examples/nas/enas/README.md) with NNI
-   * [Automatic Feature Engineering](examples/feature_engineering/auto-feature-engineering/README.md) with NNI
-   * [Hyperparameter Tuning for Matrix Factorization](https://github.com/microsoft/recommenders/blob/master/examples/04_model_select_and_optimize/nni_surprise_svd.ipynb) with NNI
-   * [scikit-nni](https://github.com/ksachdeva/scikit-nni) Hyper-parameter search for scikit-learn pipelines using NNI
-* ### **Relevant Articles** ###
-  * [Hyper Parameter Optimization Comparison](docs/en_US/CommunitySharings/HpoComparison.rst)
-  * [Neural Architecture Search Comparison](docs/en_US/CommunitySharings/NasComparison.rst)
-  * [Parallelizing a Sequential Algorithm TPE](docs/en_US/CommunitySharings/ParallelizingTpeSearch.rst)
-  * [Automatically tuning SVD with NNI](docs/en_US/CommunitySharings/RecommendersSvd.rst)
-  * [Automatically tuning SPTAG with NNI](docs/en_US/CommunitySharings/SptagAutoTune.rst)
-  * [Find thy hyper-parameters for scikit-learn pipelines using Microsoft NNI](https://towardsdatascience.com/find-thy-hyper-parameters-for-scikit-learn-pipelines-using-microsoft-nni-f1015b1224c1)
-  * **Blog (in Chinese)** - [AutoML tools (Advisor, NNI and Google Vizier) comparison](http://gaocegege.com/Blog/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0/katib-new#%E6%80%BB%E7%BB%93%E4%B8%8E%E5%88%86%E6%9E%90) by [@gaocegege](https://github.com/gaocegege) - 总结与分析 section of design and implementation of kubeflow/katib
-  * **Blog (in Chinese)** - [A summary of NNI new capabilities in 2019](https://mp.weixin.qq.com/s/7_KRT-rRojQbNuJzkjFMuA) by @squirrelsc
+## **Rleases and Contributing**
+NNI has a monthly release cycle (major releases). Please let us know if you encounter a bug by [filling an issue](https://github.com/microsoft/nni/issues/new/choose).
+
+We apprecicate all contributions. If you are planning to contribute any bug-fixes, please do so without further discussions.
+
+If you plan to contribute new features, new tuners, new training services, etc. please first open an issue or reuse an exisiting issue, and discuss the feature with us. We will discuss with you on the issue timely or set up conference calls if needed.
+
+To learn more about making a contribution to NNI, please refer to our [How-to contribution page](https://github.com/microsoft/nni/wiki/How-to-contribute). 
```

Comment:
maybe this [page](https://nni.readthedocs.io/en/v2.0/contribution.html) is more helpful.